Content.Server/Drone/DroneSystem.cs

```diff
@@ -93,12 +101,13 @@ private void OnMindAdded(EntityUid uid, DroneComponent drone, MindAddedMessage a
                 if (TryComp<HandsComponent>(uid, out var hands) && hands.Count >= drone.Tools.Count)
                 {
                    foreach (var entry in drone.Tools)
-                    {
-                        var item = EntityManager.SpawnEntity(entry.PrototypeId, spawnCoord);
-                        AddComp<UnremoveableComponent>(item);
-                        hands.PutInHand(item);
-                        drone.ToolUids.Add(item);
-                    }
+                   {
+                       var item = EntityManager.SpawnEntity(entry.PrototypeId, spawnCoord);
+                       if (item.ToString() != "ClothingBackpackSatchelDrone")
```

Comment:
same here with hardcoding, also just indent the if statement block normally

Reply:
ah yeah this was a holdover from another attempt to get the bag to drop. I'll remove this bit

Reply:
removed this